Asciidoc conversion of data movement doc (#4208)
ansible/aap-docs: downstream/modules/platform/ref-upgrade-2.5-2.6-hub.adoc

[id="ref-upgrade-2.5-2.6-hub"]

= {HubNameStart}

The following apply:

* A {PrivateHubName} admin (Automation Content Administrator) in 2.5 will be removed in the upgraded version and for this user the permissions must be reassigned manually as part of the data movement process. 

[IMPORTANT]
====
If teams with the same name exist in both {HubName} and within the platform-wide authentication gateway, users from {HubName} will be automatically added to corresponding teams within the platform-wide authentication gateway, and new teams will be created if they do not exist. 
This approach aims to retain team memberships, but requires careful review of permissions post-upgrade.
====

* If you rely on {HubName} _Single Sign-On_ (SSO) to access the {HubName} user interface (UI), {HubName} SSO logins will no longer function after the upgrade. 
However, API tokens will remain active. 
Therefore, automated processes or systems that use API tokens for authentication will continue to operate without interruption. 
If your workflows predominantly rely on API access, the impact may be minimal. 
However, if users primarily access the UI through SSO, they will need to take action post-upgrade.

> * To restore UI access for users who previously relied on automation hub SSO, you need to reconfigure SSO within {PlatformNameShort} to be able to login. 

Reviewer: Missing attribute and verb form of "log in": 
```suggestion
* To restore UI access for users who previously relied on {HubName} SSO, you need to reconfigure SSO within {PlatformNameShort} to be able to log in. 
```

> For further information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/installing_and_configuring_central_authentication_for_the_ansible_automation_platform/index#configuring-central-auth-generic-oidc-settings[Configuring Ansible Automation Platform Central Authentication Generic OIDC Settings and  Red Hat SSO/KEYCLOAK for Red Hat SSO and Ansible Automation Platform].

Reviewer: Minor nit for title (can't recall if we're required to use attributes in the title for pre-migration work; if so, replace "Ansible Automation Platform" with the attribute: 
```suggestion
For further information, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/installing_and_configuring_central_authentication_for_the_ansible_automation_platform/index#configuring-central-auth-generic-oidc-settings[Configuring Ansible Automation Platform Central Authentication Generic OIDC Settings and  Red Hat SSO/keycloak for Red Hat SSO and Ansible Automation Platform].
```

> * {ControllerName} admins will become platform admins and can administer {HubName}.

Reviewer: Minor nit, attribute start: 
```suggestion
* {ControllerNameStart} admins will become platform admins and can administer {HubName}.
```

> * If a customer upgraded from 2.4 to 2.6 with both {ControllerName} and {PrivateHubName}, then a dialog should appear in the product post upgrade that informs the user that there are steps to take in order to reconfigure {PrivateHubName}. 

Reviewer: Minor suggestion to replace "should" based on IBM style guide: 
```suggestion
* If a customer upgraded from 2.4 to 2.6 with both {ControllerName} and {PrivateHubName}, then a dialog appears in the product post upgrade that informs the user that there are steps to take in order to reconfigure {PrivateHubName}. 
```

> This dialog can either display information in-product, or link to a product doc or Knowledge Base article. In either case, the customer should be guided to take action from within the product and not be expected to find that information unprompted.

Reviewer: ```suggestion
This dialog can either display information in-product, or link to a product doc or Knowledge Base article. In either case, you will be guided to take action from within the product and not be expected to find that information unprompted.
```

> * If a customer upgraded from 2.4 to 2.6 from an {ControllerName}-only environment, then the addition of {PrivateHubName} and {EDAName} services involves adding the necessary roles to a normal user account to grant access to those services.

Reviewer: Minor suggestion
```suggestion
* If you upgraded from 2.4 to 2.6 from an {ControllerName}-only environment, then the addition of {PrivateHubName} and {EDAName} services involves adding the necessary roles to a normal user account to grant access to those services.
```
